README.yaml

```
@@ -57,42 +57,52 @@ usage: |-
   Include this module in your existing terraform code:
 
   ```hcl
+  module "label" {
+    source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.11.1"
```

```suggestion
    source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
```

```suggestion
  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref= master"
```

Pin to `master` in all examples because we will not be able to update the tags in the examples and to automate the process.
We generate a NOTE in README 

```
**IMPORTANT:** The `master` branch is used in `source` just as an example. In your code, do not pin to `master` because there may be breaking changes between releases.
Instead pin to the release tag (e.g. `?ref=tags/x.y.z`) of one of our [latest releases](https://github.com/cloudposse/terraform-aws-codebuild/releases).
```

---

ok, done